qCC/ccCommandLineCommands.cpp

```diff
@@ -139,6 +139,7 @@ constexpr char COMMAND_POP_MESHES[]						= "POP_MESHES";
 constexpr char COMMAND_NO_TIMESTAMP[]					= "NO_TIMESTAMP";
 constexpr char COMMAND_MOMENT[]							= "MOMENT";
 constexpr char COMMAND_FEATURE[]						= "FEATURE";
+constexpr char COMMAND_RGB_CONVERT_TO_SF				= "RGB_CONVERT_TO_SF"
```

Comment:
This should be 
```C++
constexpr char COMMAND_RGB_CONVERT_TO_SF[] = "RGB_CONVERT_TO_SF"
```

(the `[]` are missing)

Reply:
Oh yeah you're right